docs/content/AutoGPT/Setups/WSL2-setup.md

```diff
@@ -0,0 +1,128 @@
+### Set up with WSL2
+
+!!! important
+    Make sure you have installed a [valid distribution](https://learn.microsoft.com/en-us/windows/wsl/install)
+    Instructions here will be based on a fresh install of Ubuntu.
+
+!!! info "Executing commands"
+    To execute the given commands, open a CMD, Bash, or Powershell window.  
+    On Windows: press ++win+x++ and pick *Terminal*, or ++win+r++ and enter `cmd`
+
+1. Start your distro
+
+	```shell
+	wsl -d Ubuntu
+	```
+
+	Replace "Ubuntu" with the name of your installed distro.
+
+	Use `wsl --list` to see currently installed distributions.
+
+2. Update your distro
+
+	```shell
+	sudo apt update && sudo apt upgrade
+	```
+
+3. Install required packages
+
+	```shell
+	sudo apt-get install git python3-pip
+	```
+
+4. Create ssh keys
+
+	```shell
+	mkdir .ssh
+	ssh-keygen -t ed25519 -C "your_github_email@domain.com"
+	Enter file in which to save the key (/home/{USER}/.ssh/id_ed25519): .ssh/github
+	```
+
+5. Add ssh key to GitHub
+
+	```shell
+	cat ~/.ssh/github.pub
+	ssh-ed25519 AAAAblahblahblahblahblahblahblahblahblah your_github_email@domain.com
+	```
+
+	[Visit GitHub](https://github.com/settings/ssh/new)
+
+	Add a title e.g. WSL2 AutoGPT
+	
+	Copy the above text "ssh-ed255..." and paste it to the Key field and click "Add SSH key"
+
+6. Add the ssh key to your system
+
+	```shell
+	eval "$(ssh-agent -s)"
+	ssh-add ~/.ssh/github
+	```
+
+7. Setup git details on yout system
+
+	```shell
+	git config --global user.name "YourGitHubUsername"
+	git config --global user.email "your_github_email@domain.com"
+	```
```

Comment:
This git setup isn't necessary to use the system, and most of our current target audience already has this set up. You could replace it by a sidenote that in order to contribute to the project, Git needs to be set up, + a link to an external guide.

Reply:
@Pwuts Ah yes sorry you are right. I am also going through the instructions and they don't work.
The stable version of the repo does not have the autogpts folder. Will this change in the future?

Reply:
@Pwuts Also the instructions here: https://docs.agpt.co/autogpt/setups/git-setup/ are also wrong because the autogpts folder does not exist. This actually reminded me, this was another issue I had starting out as downloading the latest stable version, there is no .env.template file to be found anywhere, I believe I only fluked it that I tried cloning again without reading the instructions and therefore ended up with the master version which worked.

Reply:
Thats a good point actually, the documentation refer to the master branch already and we didn't update the stable branch in a very long time now.